src/crypto/secp256k1/ecdsa/ecdsa.cairo

```diff
@@ -0,0 +1,57 @@
+from starkware.cairo.common.uint256 import Uint256
+
+from starkware.cairo.common.cairo_secp.signature import get_point_from_x
+from starkware.cairo.common.cairo_secp.bigint import BigInt3, uint256_to_bigint
+from starkware.cairo.common.cairo_secp.ec import EcPoint, ec_add, ec_mul
+from starkware.cairo.common.cairo_secp.signature import (
+    validate_signature_entry,
+    get_generator_point,
+    div_mod_n,
+)
+
+func get_ecpoint_from_pubkey{range_check_ptr}(x: Uint256, y: Uint256) -> (ec : EcPoint) {
```

Comment:
Unary tupels as a return type are too verbose. Simply return the plain `EcPoint` here.